cmake/AddPlaydateApplication.cmake

```diff
@@ -47,48 +53,46 @@ function(add_playdate_application PLAYDATE_GAME_NAME)
                     TARGET ${PLAYDATE_GAME_NAME} POST_BUILD
                     COMMAND ${CMAKE_COMMAND} -E copy
                     ${CMAKE_CURRENT_BINARY_DIR}/${BUILD_SUB_DIR}${PLAYDATE_GAME_NAME}.dll
-                    ${CMAKE_CURRENT_SOURCE_DIR}/Source/pdex.dll)
+                    ${PDCPP_STAGING_DIR}/Source/pdex.dll)
         elseif (MINGW)
             add_custom_command(
                     TARGET ${PLAYDATE_GAME_NAME} POST_BUILD
                     COMMAND ${CMAKE_COMMAND} -E copy
                     ${CMAKE_CURRENT_BINARY_DIR}/lib${PLAYDATE_GAME_NAME}.dll
-                    ${CMAKE_CURRENT_SOURCE_DIR}/Source/pdex.dll)
+                    ${PDCPP_STAGING_DIR}/Source/pdex.dll)
         elseif(APPLE)
             if(${CMAKE_GENERATOR} MATCHES "Xcode" )
                 set(BUILD_SUB_DIR $<CONFIG>/)
-                set_property(TARGET ${PLAYDATE_GAME_NAME} PROPERTY XCODE_SCHEME_ARGUMENTS \"${CMAKE_CURRENT_SOURCE_DIR}/${PLAYDATE_GAME_NAME}.pdx\")
+                set_property(TARGET ${PLAYDATE_GAME_NAME} PROPERTY XCODE_SCHEME_ARGUMENTS \"${PDCPP_STAGING_DIR}/${PLAYDATE_GAME_NAME}.pdx\")
                 set_property(TARGET ${PLAYDATE_GAME_NAME} PROPERTY XCODE_SCHEME_EXECUTABLE ${SDK}/bin/Playdate\ Simulator.app)
             endif()
 
             add_custom_command(
                 TARGET ${PLAYDATE_GAME_NAME} POST_BUILD
                 COMMAND ${CMAKE_COMMAND} -E copy
                 ${CMAKE_CURRENT_BINARY_DIR}/${BUILD_SUB_DIR}lib${PLAYDATE_GAME_NAME}.dylib
-                ${CMAKE_CURRENT_SOURCE_DIR}/Source/pdex.dylib)
+                ${PDCPP_STAGING_DIR}/Source/pdex.dylib)
 
         elseif(UNIX)
             add_custom_command(
                 TARGET ${PLAYDATE_GAME_NAME} POST_BUILD
                 COMMAND ${CMAKE_COMMAND} -E copy
                 ${CMAKE_CURRENT_BINARY_DIR}/lib${PLAYDATE_GAME_NAME}.so
-                ${CMAKE_CURRENT_SOURCE_DIR}/Source/pdex.so)
+                ${PDCPP_STAGING_DIR}/Source/pdex.so)
         else()
             message(FATAL_ERROR "Platform not supported!")
         endif()
 
         set_property(
             TARGET ${PLAYDATE_GAME_NAME} PROPERTY ADDITIONAL_CLEAN_FILES
-            ${CMAKE_CURRENT_SOURCE_DIR}/${PLAYDATE_GAME_NAME}.pdx
+            ${PDCPP_STAGING_DIR}/${PLAYDATE_GAME_NAME}.pdx
         )
 
         add_custom_command(
             TARGET ${PLAYDATE_GAME_NAME} POST_BUILD
-            COMMAND ${PDC} ${CMAKE_CURRENT_SOURCE_DIR}/Source
-            ${CMAKE_CURRENT_SOURCE_DIR}/${PLAYDATE_GAME_NAME}.pdx)
+            COMMAND ${PDC} ${PDCPP_STAGING_DIR}/Source
+            ${PDCPP_STAGING_DIR}/${PLAYDATE_GAME_NAME}.pdx)
     endif()
 
-
-    target_compile_options(${PLAYDATE_GAME_NAME} PUBLIC $<$<CONFIG:DEBUG>:-O0>)
-    target_compile_options(${PLAYDATE_GAME_NAME} PUBLIC $<$<CONFIG:RELEASE>:-O3>)
-endfunction()
\ No newline at end of file
+    target_link_libraries(${PLAYDATE_GAME_NAME} PUBLIC pdcpp_core)
```

Comment:
Adding this linkage with explicit linkage type breaks the examples because they don't have that. There's no problem with the addition, just go fix the examples so they're not misleading to new folks.

Reply:
Fixed!